src/templates/CommitteeTemplate.jsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import Markdown from 'markdown-to-jsx';
+import styled from 'styled-components';
+import { graphql } from 'gatsby';
+import Layout from '../components/layout/Layout';
+import { Grid, Header, Image, List, Button, Icon } from 'semantic-ui-react';
+
+const CommitteeView = ({ data }) => {
+  const { contentfulCommittee: committee } = data;
+  const photo = committee.photo != null;
+
+  return (
+    <Layout>
+      <Button className="labeled icon basic" href="/vereniging/commissies">
+        <Icon name="angle left" />
+        Commissies
+      </Button>
+      <Grid columns={3} style={{ 'margin-top': '5px' }} stackable>
```

Comment:
You used a lot of Grid components, which can be just regular divs and styled with CSS-grid. I prefer the CSS-Grid way, as this is also consistent with the rest of the grids in the code...

Reply:
Actually, for responsive grids I prefer the components from semantic ui. They have a very nice 'doubling' prop, which makes them responsive without us having to write media queries all the time.

Reply:
We talked about this in a meeting and we decided that for grids with homogeneous items (like the activity index), but for grids with heterogeneous items (like these) we should use CSS-Grid, because of the `grid-template-areas` feature, which allows for simple configuration of complex layouts.